cnxepub/html_parsers.py

```diff
@@ -197,18 +201,34 @@ def keywords(self):
 
     @property
     def license_url(self):
-        items = self.parse('.//xhtml:*[@data-type="license"]/@href')
+        # Three cases for location of metadata stanza:
+        #  1. direct child of current node
+        #  2. direct child of any ancestor
+        #  3. Top of book (occurs when fetching from root)
+        items = self.parse('ancestor-or-self::*/*[@data-type="metadata"]//*'
+                           '[@data-type="license"]/@href'
+                           ' | /xhtml:html/xhtml:body/*'
```

Comment:
isn't `xhtml:body` one of the `ancestor`'s already caught by the previous XPath?